denguefever_tw/dengue_linebot/models.py

```diff
@@ -40,9 +40,9 @@ class MinArea(models.Model):
     area = models.PolygonField(srid=4326)
 
     def __str__(self):
-        return ' {name} ({id})'.format(
-            name=self.area_name,
-            id=self.area_id
+        return ' {dis} {area}'.format(
+            dis=self.district_name,
```

Comment:
Use district as key would be better. dis is usually used as the abbreviation of distance.